ddcommon-ffi/src/string.rs

```diff
@@ -72,3 +73,45 @@ pub unsafe extern "C" fn ddog_StringWrapper_message(s: Option<&StringWrapper>) -
         Some(s) => CharSlice::from(s.as_ref()),
     }
 }
+
+pub trait ToHexStr {
+    fn to_hex_str(&self) -> String;
+}
+
+impl ToHexStr for usize {
+    fn to_hex_str(&self) -> String {
+        format!("0x{:X}", self)
```

Comment:
Do we want this to be fixed length? i.e. should `2` be `0x2` or `0x0000000000000002`?

Reply:
🤷 
the specification says that it's a hexadecimal string prefixed with `0x` but nothing about the length.

Reply:
Agreed that either is standards compliant.  Its just aesthetically, which do we prefer?